Adyen/Helpers/UIViewHelpers.swift

```diff
@@ -19,4 +20,20 @@ extension AdyenScope where Base: UIView {
 
         return image
     }
+
+    /// :nodoc:
+    public func getTopMostView<T: UIView>() -> T? {
+        var queue = [UIView]()
```

Comment:
can be simplified as `var queue = [base]`

Reply:
👍